src/Sylius/Bundle/AdminBundle/Resources/views/Taxon/_form.html.twig

```diff
@@ -2,7 +2,11 @@
     {{ form_errors(form) }}
     <div class="two fields">
         {{ form_row(form.code) }}
-        {{ form_row(form.parent) }}
+        {% if taxon.id != null %}
+            {{ form_row(form.parent) }}
+        {% else %}
+            {{ form_widget(form.parent, {'attr': {'style': 'display: none;'}}) }}
```

Comment:
Yup, but what if we want to change parent of existing taxon? Or is it the feature we don't want to support?